ntoskrnl/mm/ARM3/section.c

```diff
@@ -3619,16 +3624,17 @@ NtOpenSection(OUT PHANDLE SectionHandle,
 
 NTSTATUS
 NTAPI
-NtMapViewOfSection(IN HANDLE SectionHandle,
-                   IN HANDLE ProcessHandle,
-                   IN OUT PVOID* BaseAddress,
-                   IN ULONG_PTR ZeroBits,
-                   IN SIZE_T CommitSize,
-                   IN OUT PLARGE_INTEGER SectionOffset OPTIONAL,
-                   IN OUT PSIZE_T ViewSize,
-                   IN SECTION_INHERIT InheritDisposition,
-                   IN ULONG AllocationType,
-                   IN ULONG Protect)
+NtMapViewOfSection(
+    _In_ HANDLE SectionHandle,
+    _In_ HANDLE ProcessHandle,
+    _Outptr_result_bytebuffer_(*ViewSize) _Pre_valid_ PVOID *BaseAddress,
+    _In_ ULONG_PTR ZeroBits,
+    _In_ SIZE_T CommitSize,
+    _Inout_opt_ PLARGE_INTEGER SectionOffset,
+    _Inout_ PSIZE_T ViewSize,
+    _In_range_(ViewShare, ViewUnmap) SECTION_INHERIT InheritDisposition,
+    _In_ ULONG AllocationType,
+    _In_ ULONG Win32Protect)
```

Comment:
For information (here and the other changes below): This last parameter was named "Protect" in the Nt/ZwMapViewOfSection declaration in Windows 2000 and XP DDK ntddk.h and wdm.h headers, but has been renamed "Win32Protect" in the Windows 2003 (".net") DDK headers.

What do we want to do then? Change the name? (why "Win32" has been prepended to "Protect"? Is it because there is a translation to "non-Win32" values?) Or keep the original name?
(This is a question maybe more for @tkreuzer and @ThFabba )

Reply:
I changed it to `Win32Protect` per
https://learn.microsoft.com/en-us/windows-hardware/drivers/ddi/wdm/nf-wdm-zwmapviewofsection
and because
https://github.com/reactos/reactos/blob/726c662242960ec9976be0bff17cc5fd1448bc72/sdk/include/ndk/mmfuncs.h#L411
has it like that already.

I could sync it the other way round, if you prefer.

Reply:
It is 100% irrelevant.

Reply:
If it is irrelevant, then let's use the shorter version (keep it like it was), because it will also minimize the touched lines in this PR.